src/Illuminate/Queue/Queue.php

```diff
@@ -188,4 +195,27 @@ public function setContainer(Container $container)
     {
         $this->container = $container;
     }
+
+    /**
+     * Set the queue prefix.
+     *
+     * @param  string  $prefix
+     * @return $this
+     */
+    public function setQueuePrefix($prefix = null)
```

Comment:
I think there was a method on a parent or child class named similarly, so someone suggested it to help reduce ambiguity.